tannerb9/MasksNowTN-New
# Tb-info

# Description
Added the two paragraphs to the left column and the 'We Can Do It' image to the right column of the Info section(have not added styling or responsiveness on top of Vuetify's built-ins)

## Type of change

# Testing Instructions
Run the app and ensure the title(Masks Now TN), two paragraphs, and image display in the Info section(first section under Navbar).
